src/internalHelpers/_parseColorString.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+/** */
+type RgbColor = {
+  red: number,
+  green: number,
+  blue: number,
+  alpha?: number,
+}
+
+const hexRegex = new RegExp('^#[a-fA-F0-9]{6}$')
+const reducedHexRegex = new RegExp('^#[a-fA-F0-9]{3}$')
+// eslint-disable-next-line no-useless-escape
+const rgbaRegex = /^rgba\((\d+),(\d+),(\d+),([-+]?[0-9]*[.]?[0-9]+)\)$/
+// const rgbaRegex = new RegExp('^rgba\((\d+),(\d+),(\d+),([-+]?[0-9]*[.]?[0-9]+)\)$')
```

Comment:
Ahh, good catch!
First I used `new RegExp` syntax, but there linting complained, but the escapes were not useless. They were needed. With `/<regex>/` syntax it's detected properly … let's use this and remove the eslint disable